apps/osim/workflow.py

```diff
@@ -257,3 +257,24 @@ def promote(self, save=True, jira_token=None):
             return
 
         self.save(jira_token=jira_token)
+
+    def reject(self, save=True, jira_token=None):
+        """
+        this is the cannonical way of rejecting a flaw / task
+
+        This method will change instance state to rejected if all conditions are met
+
+        Raise exception if instance lacks requirements for the rejected state
+        """
+        reject_workflow = "NOT-AFFECTED"
```

Comment:
I see that you are reusing the example NOT-AFFECTED workflow. I do not see that you would be changing it. There are a lot of issues with that. Simply starting with the name, it is clearly wrong. We should have some REJECTED workflow instead. Then description of the workflow is about something completely different. The conditions are unnecessary for the flaw draft just being rejected. There are three states while it is not clear why we should have more then one. There is no state REJECTED in that workflow. This workflow should be just dropped and not reused. It seems that the logic is already far from the original **osim** module aim and we will not have different workflows for different flaw contexts (like Major Incident, not affected, embargoed etc.) as originally intended. It seems that now it only makes sense to have DEFAULT and REJECTED.